README.md

```diff
@@ -24,7 +24,7 @@ grunt.loadNpmTasks('grunt-contrib-qunit');
 ## Qunit task
 _Run this task with the `grunt qunit` command._
 
-You have chosen to write your unit tests using [QUnit][], you have written a
+You have chosen to write your unit tests using [QUnit](http://qunitjs.com/), you have written a
```

Comment:
Assuming you have already updated the file `docs/qunit-overview.md` and you run `./node_modules/.bin/grunt` to generate the new `README`.

If you run `git status` you will get an output similar to the following:
```bash
On branch patch-1
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   README.md
	modified:   docs/qunit-overview.md

no changes added to commit (use "git add" and/or "git commit -a")
```

If that the case, then you have to add your files in order to be committed. To do so, you can either run:
```bash
git add .
# Or
git add -A
```

Then commit them as @leobalter mentioned in his comment.